src/filters/utils.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module paste-from-office/filters/utils
+ */
+
+/* globals btoa */
+
+/**
+ * Converts given HEX string to base64 representation.
+ *
+ * @param {String} hexString The HEX string to be converted.
+ * @returns {String} Base64 representation of a given HEX string.
+ */
+export function convertHexToBase64( hexString ) {
```

Comment:
Does this util have to be public? It's used in one place. If you want to test it, export it from the file where it's used as an underscored, protected util.